### DISCUSSION TITLE
https://www.spsanderson.com/steveondata/posts/2024-12-27/

### DISCUSSION THREAD

**Discussion by giscus** [spsanderson/steveondata#20] [Comments]
# https://www.spsanderson.com/steveondata/posts/2024-12-27/

Learn how to efficiently search for files in Linux using powerful commands like find and locate. A comprehensive guide for beginners with practical examples.

https://www.spsanderson.com/steveondata/posts/2024-12-27/

**Comment by gitvectors** [comment]
On Ubuntu I use Recoll indexing engine for complex searching of multiple MIME types. It can be used as GUI or CLI.
recollq --help
In GUI hover cursor over query field to see search operators .. cheat sheet.
